src/navigation/RootNavigator.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
love the comments in this file!! really helps with understanding the huge chunks of code :D